### DISCUSSION TITLE
Code Review IM-11

### DISCUSSION THREAD

**Discussion by volodymyrvikulin** [HowProgrammingWorks/LiveQA#56] [Code review]
Пишу браузерную игру "Car Racing":
Игра для двоих игроков с одной клавиатуры, задача игроков продержаться как можно дольше, уворачиваясь от вражеских машин, побеждает тот кого последним собьют. В будущем планирую добавить систему жизней и раундов, чтоб было несколько заездов. Буду благодарен за ревью [проекта](https://github.com/vladimirvikulin/Car-Racing)

**Comment by tshemsedinov** [comment]
First review https://github.com/vladimirvikulin/Car-Racing/issues/1

**Comment by volodymyrvikulin** [comment]
Все відрефакторив та додав нового функціоналу, буду вдячний за друге ревью [проекта](https://github.com/vladimirvikulin/Car-Racing)
